src/check/parse/tokenize.zig

```diff
@@ -1083,25 +1091,15 @@ pub const Tokenizer = struct {
         const start = self.cursor.pos;
         // Skip the initial quote.
         self.cursor.pos += 1;
-        var kind: StringKind = .single_line;
+        var multiline: bool = false;
         if (self.cursor.peek() == term and self.cursor.peekAt(1) == term) {
             self.cursor.pos += 2;
-            kind = .multi_line;
+            multiline = true;
         }
-        return try self.tokenizeStringLikeLiteralBody(.start, kind, term, start);
+        return try self.tokenizeStringLikeLiteralBody(false, term, start, multiline);
     }
 
-    const StringState = enum {
-        start,
-        after_interpolation,
-    };
-
-    const StringKind = enum {
-        single_line,
-        multi_line,
-    };
-
-    pub fn tokenizeStringLikeLiteralBody(self: *Tokenizer, state: StringState, kind: StringKind, term: u8, start: u32) !Token.Tag {
+    pub fn tokenizeStringLikeLiteralBody(self: *Tokenizer, already_started: bool, term: u8, start: TokenIndex, multiline: bool) !Token.Tag {
```

Comment:
This does look suspiciously like you based this on a slightly earlier version of my tokenizer (not the one that landed, after I implemented several fixup suggestions)

Reply:
I'll take a look